local-proxy/src/server.ts

```diff
@@ -25,7 +27,23 @@ if (!isAbsolute(basePath)) {
   throw new Error('SHIFT_DEV_SERVER_BASE_REQUIRE_PATH env var is not an absolute path');
 }
 
-function setupEnv({ port }: { port: number }) {
+async function loadDotEnv() {
+  const envFile = resolvePath(basePath, '.env');
```

Comment:
Doesn't seem like it's the right path.

Reply:
Why not?
It does work.. there's a test for it.

Reply:
Sorry for eliding the point and being unclear.  This is the same path as for deploying.  That _might_ be the right env, but I think I'd more often want *different* config values when running locally.  With this, I get a _separate_ reshuffle DB when I run locally but the _same_ e.g. RedisLabs DB.

Reply:
Maybe we should re-review. An `.env` file in [heroku local](https://devcenter.heroku.com/articles/heroku-local#copy-heroku-config-vars-to-your-local-env-file) behaves in the opposite way (only affects local environment and not the remote app.